libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_api.c

```diff
@@ -645,6 +790,13 @@ TEST( MQTT_Unit_API, OperationWaitTimeout )
 
     /* An arbitrary MQTT packet for this test. */
     static uint8_t pPacket[ 2 ] = { MQTT_PACKET_TYPE_PINGREQ, 0x00 };
+    bool subscriptionMutexCreated = false;
+    bool contextMutex = false;
+    TransportInterface_t transport;
+    MQTTFixedBuffer_t networkBuffer;
+    MQTTApplicationCallbacks_t callbacks;
+    MQTTStatus_t managedMqttStatus = MQTTBadParameter;
+    int8_t contextIndex = -1;
```

Comment:
These variables doesn't seems to be used. Please remove those

Reply:
Addressed